doc/connection.rst

```diff
@@ -119,7 +119,7 @@ Connection Types
 
 .. class:: SshConnection(host, username, password=None, keyfile=None, port=None,\
                          timeout=None, password_prompt=None, \
-                         sudo_cmd="sudo -- sh -c {}")
+                         sudo_cmd="sudo -- sh -c {}, options=None")
```

Comment:
Shouldn't the `, options=None` be ouside the `"`?

Reply:
Oops, yes.